### PR TITLE
release notes

### PR DESCRIPTION
**Pull request by jamespohalloran**
Release notes for 1.3.0

### CODE REVIEW

File: content/blog/tina-v-1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+date: '2023-03-08T04:00:00.000Z'
+last_edited: '2023-03-08T04:00:00.000Z'
+author: James O'Halloran
+---
+
+TinaCMS Version `1.3.0` (and `@tinacms/cli@1.2.1`) improves media, adds support for special characters in field names, and some other fixes and improvements.
+
+## Using Field names with special characters (using nameOverride)
+
+Tina now supports using fields special characters (or using other special field names like `id`) through the use of the `nameOverride` property.
```
Comment (kldavis4) [inline content/blog/tina-v-1.3.0.md:12]: not parsing this

File: content/blog/tina-v-1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: TinaCMS V1.3.0
+date: '2023-03-08T04:00:00.000Z'
+last_edited: '2023-03-08T04:00:00.000Z'
+author: James O'Halloran
+---
+
+TinaCMS Version `1.3.0` (and `@tinacms/cli@1.2.1`) improves media, adds support for special characters in field names, and some other fixes and improvements.
```
Comment (logan-anderson) [inline content/blog/tina-v-1.3.0.md:8]: Its @tinacms/cli@1.3.1  not 1.2.1

File: content/blog/tina-v-1.3.0.md (hunk 1 above)
Comment (jamespohalloran) [reply content/blog/tina-v-1.3.0.md]: Made a tweak, that any better?

**Review by kldavis4 — APPROVED** [review]

File: content/blog/tina-v-1.3.0.md (hunk 1 above)
Comment (kldavis4) [reply content/blog/tina-v-1.3.0.md]: looks good

**Pull request merged**